jxls/src/main/java/org/jxls/area/CommandData.java

```diff
@@ -31,20 +31,20 @@ public CommandData(String areaRef, Command command) {
 
     public CommandData(CellRef startCellRef, Size size, Command command) {
         this.startCellRef = startCellRef;
-        this.size = size;
+        this.commandSize = size;
         this.command = command;
     }
 
     public AreaRef getAreaRef() {
-        return new AreaRef(startCellRef, size);
+        return new AreaRef(startCellRef, commandSize);
     }
 
     public CellRef getStartCellRef() {
         return startCellRef;
     }
 
-    public Size getSize() {
-        return size;
+    public Size getCommandSize() {
```

Comment:
Renaming public methods is a breaking change, which means we will have to bump a major Jxls version and release Jxls 3.0.0. But releasing a new major version probably is not worth it unless we are ready also to  introduce new major features.